src/Controls/tests/TestCases.HostApp/Issues/Issue14825.xaml.cs

```diff
@@ -0,0 +1,31 @@
+﻿#nullable enable
+namespace Maui.Controls.Sample.Issues;
+
+[XamlCompilation(XamlCompilationOptions.Compile)]
+[Issue(IssueTracker.Github, 14825, "Capture WebView screenshot", PlatformAffected.UWP)]
+public partial class Issue14825 : ContentPage
+{
+	public Issue14825()
+	{
+		InitializeComponent();
+	}
+
+	private async void CaptureButton_Clicked(object sender, EventArgs e)
+	{
+		IScreenshotResult? result = await myWebView.CaptureAsync();
+
+		if (result != null)
+		{
+			// Intentionally no "using" because ImageSource requires a valid stream.
+			Stream stream = await result.OpenReadAsync(ScreenshotFormat.Png, 100);
+
+			screenshotResult.Add(new Label() { Text = $"Your screenshot ({result.Width}x{result.Height}):" });
+
+			DisplayInfo displayInfo = DeviceDisplay.MainDisplayInfo;
```

Comment:
Is it possible not to just make the image view and webview the same size initially? I am not a fan of using the `DeviceDisplay.MainDisplayInfo` because it does not handle multi-monitor and sometimes if a popup appears then the main display could switch.

Might be worth having the image view there in the xaml with the same size and then just set the source.